metabase/metabase
# Follow-up milestone 2 for [Epic] Re-design transparent theme for static embedding and public links

Relate to Epic #43838

### Description

This PR accomplishes 3 things.
1. Introduce [new base colors](https://www.figma.com/design/OlyKP09vro7EkQ2ZzXDDla/color-playground?node-id=501-2549&t=S7YjATSbqaPbQKuZ-0)
2. Introduce hover colors. ([Github comment](https://github.com/metabase/metabase/pull/46428#discussion_r1702281088)). This makes many components' hover and selected state consistent.
3. Fix [embed frame border color](https://github.com/metabase/metabase/pull/46428#pullrequestreview-2216037341)

### How to verify

1. View all new loki snapshots, and you should see that the hover/selected colors are more consistent.
2. Embed the dashboard/question in another application and the border shouldn't look off.

### Demo

View Loki snapshots

### Checklist

- [x] Tests have been added/updated to cover changes in this PR (loki snapshots)
